### PR TITLE
Fixing negative lineshapes

### PR DESCRIPTION
**Pull request by dcmvdbekerom**
### Description

This pull request is to address https://github.com/radis/radis/issues/186.

With the new folding procedure the FFT lineshapes behave the same way as the conventional lineshapes when the linewidth is approximately equal to wstep.

The negative lineshapes are still an issue when `min-RMS` is used as optimization scheme, leading me to believe that it should _not_ be used as the default.

Fixes #<Issue Number>

### CODE REVIEW

File: radis/lbl/broadening.py [hunk 1]
```diff
@@ -621,6 +611,34 @@ def voigt_lineshape(w_centered, hwhm_lorentz, hwhm_voigt, jit=True):
     return lineshape
 
 
+def voigt_FT(w_lineshape_ft, hwhmG, hwhmL):
+    """Fourier Transform of a Lorentzian lineshape
```
Comment (erwanp) [inline radis/lbl/broadening.py:615]:
```suggestion
    """Fourier Transform of a Voigt lineshape
```
No?

**Review by erwanp — APPROVED** [review]
Nice! 

Is there any particular reason you inverted the nested loops  Gaussian first,  then Lorentzian during the database filling?

**Comment by dcmvdbekerom** [comment]
> Is there any particular reason you inverted the nested loops Gaussian first, then Lorentzian during the database filling?

No particular reason, other than that that's what I'm used to, and also what I used in the paper and my [lightweight code](https://github.com/dcmvdbekerom/discrete-integral-transform/blob/master/demo/discrete_integral_transform.py)

Back in the days I got confused with the order so often that I settled on G,L because it is alphabetic. I kept this order to make it easier to drop in the working code, and as an additional benefit it made it easier to see which of the RADIS code I had checked and which I hadn't.

Long story short, the G,L order was just for my own convenience, we can easily keep it L,G like it was before.

**Comment by erwanp** [comment]
Both options are fine. Merging!
